packages/ui/src/theme/tokens/types/scales.ts

```diff
@@ -1,14 +1,14 @@
 import { DesignToken } from './designToken';
 
-export interface OrdinalScale {
-  primary: DesignToken;
-  secondary: DesignToken;
-  tertiary: DesignToken;
+export interface OrdinalScale<ValueType = any> {
```

Comment:
can we use a more constrained set of value types? (rather than any)

Reply:
Good call, it should be ColorValue